device/mellanox/x86_64-mlnx_lssn2700-r0/plugins/sfputil.py

```diff
@@ -64,6 +92,21 @@ def get_presence(self, port_num):
             return True
 
         return False
+        
+    def get_presence_status_file_path(self, port_num):
```

Comment:
>get_presence_status_file_path [](start = 8, length = 29)

Who is calling this function?

Reply:
true, not called anymore, call in the original solution. removed.